app/views/products/new.html.erb

```diff
@@ -0,0 +1,13 @@
+
+<h2>Create a New Product</h2>
+
+<%= form_for [@vendor, @product] do |f| %>
+	<%= f.label :'Product Name:' %>
+	<br>
+	<%= f.text_field :name, class: 'field' %>
+	<%= f.hidden_field(:vendor_id, :value => @vendor.id) %>
+	<br>
+<div class="product_button">
+	<%= f.submit class: 'btn btn-success'%>
+</div>
+<% end %>
```

Comment:
You can use the same strategy as before to reuse your form between products/new and products/edit